algorithm.go

```diff
@@ -4,13 +4,13 @@ import "fmt"
 
 const (
 	// HmacSha1 describes a HMAC with SHA-1
-	HmacSha1 Algorithm = 0x01
```

Comment:
Not sure about this - this is already a public constant. What good does the additional constant do?